<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectScreen</class>
 <widget class="QDialog" name="ObjectScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Novo Objeto</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="lbl_desc_transformacao">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Adicione as transformações do objeto</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_coord_x">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add_transformation">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Adicionar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_coord_y">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_coord_x">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string extracomment="Testes"/>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_coord_y">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">Testes</string>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbtn_escalonar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Escalonar</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="btn_transladar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Transladar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_coord_z">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>z</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_coord_z">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">Testes</string>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>241</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_list_transformations">
    <item>
     <widget class="QListView" name="list_transformations"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_apply_transformations">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>490</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Aplicar</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbtn_rotacao_centro_mundo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Em torno do centro do mundo</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_rotacionar">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rotacionar</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbtn_rotacao_centro_objeto">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Em torno do centro do objeto</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbtn_rotacao_ponto">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>321</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Em torno de um ponto qualquer (arbitrário)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_angulo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ângulo:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_angulo_rotacao">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">Testes</string>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_ponto">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ponto</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_point">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>460</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">Testes</string>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add_rotacao">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Adicionar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clear_list_transformation">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>520</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Limpar Lista</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
